src/index.ts

```diff
@@ -622,7 +623,19 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
     eventSourceWithMiddleware = addMiddlewareToEventSource(middleware)
   }
 
-  return { on, off, variation, close, setEvaluations, registerAPIRequestMiddleware }
+  const refreshCache = (): boolean => {
+    // only fetch flags if enough time has elapsed to avoid pressuring backend servers
+    if (lastCacheRefreshTime == null || Math.round((Date.now()-lastCacheRefreshTime)/1000) >= 60) {
```

Comment:
In JS you should always use type-safe comparisons (`===`, `>==` etc). Also no need to use `== null`. A falsey check is fine
```suggestion
    if (!lastCacheRefreshTime || Math.round((Date.now() - lastCacheRefreshTime) / 1000) >== 60) {
```